bedrock/mozorg/redirects.py

```diff
@@ -659,4 +659,12 @@ def decider(request, **kwargs):
 
     # Bug 1384370
     redirect(r'^developers/?$', 'mozorg.developer'),
+
+    # Bug 1428150
+    gone(r'^tabzilla/transbar\.jsonp$'),
+    gone(r'^tabzilla/tabzilla\.js$'),
+    gone(r'^tabzilla/media/js/tabzilla\.js$'),
+    redirect(r'tabzilla/media/css/tabzilla\.css$',
```

Comment:
Is it worth having a test for this redirect? (or any of the 410's above?)

Reply:
Not sure. Probably not...

Reply:
I've added tests for all of this